projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -798,8 +798,7 @@ public void initTestrig(
     // Create metadata file (RELPATH_METADATA_FILE is "metadata.json")
     Path metadataPath = testrigDir.resolve(BfConsts.RELPATH_METADATA_FILE);
     long time = (new java.util.Date()).getTime();
-    CommonUtil.writeFile(metadataPath, "{\n\t\"timestamp\": "+Long.toString(time)+"\n}");
-
+    CommonUtil.writeFile(metadataPath, "{\n\t\"timestamp\": " + Long.toString(time) + "\n}");
```

Comment:
Don't know why I didn't have to reformat this file before merging the PR that introduced it, but travis wasn't having any of it this time